### PR TITLE
[minor_change] Change the attribute type of children that only allow one child to be configured from a set of maps (SetNestedAttribute) to single map (SingleNestedAttribute) input (DCNE-172)

### PR DESCRIPTION
**Pull request by akinross**

### CODE REVIEW

**Review by gmicol — DISMISSED** [review]
LGTM

**Review by sajagana — DISMISSED** [review]
LGTM!

**Review by samiib — APPROVED** [review]
LGTM

**Review by shrsr — APPROVED** [review]
LGTM

File: docs/resources/vrf_fallback_route_group.md [hunk 1]
```diff
@@ -121,8 +119,7 @@ All examples for the VRF Fallback Route Group resource can be found in the [exam
   * `name` (name) - (string) The name of the VRF Fallback Route Group Member object.
   * `name_alias` (nameAlias) - (string) The name alias of the VRF Fallback Route Group Member object.
 
-* `vrf_fallback_routes` - (list) A list of VRF Fallback Routes (ACI object [fvFBRoute](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvFBRoute/overview)). VRF Fallback Routes can also be configured using a separate [aci_vrf_fallback_route](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/vrf_fallback_route) resource.
-    - Max Items: 1
+* `vrf_fallback_route` - (map) A map of VRF Fallback Route (ACI object [fvFBRoute](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvFBRoute/overview)). VRF Fallback Route can also be configured using a separate [aci_](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/) resource.
```
Comment (lhercot) [inline docs/resources/vrf_fallback_route_group.md:122]: missing the full name of the resource it is only "aci_"

**Review by lhercot — CHANGES_REQUESTED** [review]

File: docs/resources/vrf_fallback_route_group.md (hunk 1 above)
Comment (akinross) [reply docs/resources/vrf_fallback_route_group.md]: fixed

**Review by samiib — COMMENTED** [review]
Are these changes backwards compatible? If not this looks to me like a breaking change and can use `major_change` in the commit message instead.

**Review by samiib — APPROVED** [review]
LGTM

File: docs/data-sources/l3out_consumer_label.md [hunk 2]
```diff
@@ -58,11 +58,11 @@ data "aci_l3out_consumer_label" "example_l3_outside" {
 * `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
 * `tag` (tag) - (string) Specifies the color of a policy label.
 
-* `relation_to_external_epgs` - (list) A list of Relation From L3Out Consumer Label To External EPGs (ACI object [l3extRsLblToInstP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extRsLblToInstP/overview)) pointing to External EPG (ACI Object [l3extInstP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extInstP/overview)). This attribute is supported in ACI versions: 5.0(1k) and later.
+* `relation_from_l3out_consumer_label_to_external_epgs` - (list) A list of Relation From L3Out Consumer Label To External EPGs (ACI object [l3extRsLblToInstP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extRsLblToInstP/overview)) pointing to External EPG (ACI Object [l3extInstP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extInstP/overview)). This attribute is supported in ACI versions: 5.0(1k) and later.
```
Comment (sajagana) [inline docs/data-sources/l3out_consumer_label.md:61]: This needs to be reverted.

File: docs/data-sources/l3out_consumer_label.md [hunk 3]
```diff
@@ -58,11 +58,11 @@ data "aci_l3out_consumer_label" "example_l3_outside" {
...
   * `target_dn` (tDn) - (string) The distinguished name (DN) of the External EPG object.
 
-* `relation_to_route_control_profiles` - (list) A list of Relation From L3Out Consumer Label To Route Control Profiles (ACI object [l3extRsLblToProfile](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extRsLblToProfile/overview)) pointing to Route Control Profile (ACI Object [rtctrlProfile](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/rtctrlProfile/overview)). This attribute is supported in ACI versions: 5.0(1k) and later.
+* `relation_from_l3out_consumer_label_to_route_control_profiles` - (list) A list of Relation From L3Out Consumer Label To Route Control Profiles (ACI object [l3extRsLblToProfile](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extRsLblToProfile/overview)) pointing to Route Control Profile (ACI Object [rtctrlProfile](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/rtctrlProfile/overview)). This attribute is supported in ACI versions: 5.0(1k) and later.
```
Comment (sajagana) [inline docs/data-sources/l3out_consumer_label.md:65]: This needs to be reverted.

File: docs/data-sources/netflow_exporter_policy.md [hunk 4]
```diff
@@ -71,12 +71,12 @@ data "aci_netflow_exporter_policy" "example_tenant" {
 * `source_ip_address` (srcAddr) - (string) The source IP address.
 * `version` (ver) - (string) The NetFlow Exporter Version of the NetFlow Exporter Policy object.
 
-* `relation_to_vrf` - (list) A list of Relation From NetFlow Exporter To VRF (ACI object [netflowRsExporterToCtx](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowRsExporterToCtx/overview)) pointing to VRF (ACI Object [fvCtx](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvCtx/overview)).
+* `relation_from_netflow_exporter_to_vrf` - (map) A map of Relation From NetFlow Exporter To VRF (ACI object [netflowRsExporterToCtx](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowRsExporterToCtx/overview)) pointing to VRF (ACI Object [fvCtx](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvCtx/overview)).
```
Comment (sajagana) [inline docs/data-sources/netflow_exporter_policy.md:74]: This needs to be reverted.

File: docs/data-sources/netflow_exporter_policy.md [hunk 5]
```diff
@@ -71,12 +71,12 @@ data "aci_netflow_exporter_policy" "example_tenant" {
...
   * `annotation` (annotation) - (string) The annotation of the Relation From NetFlow Exporter To VRF object.
   * `target_dn` (tDn) - (string) The distinguished name of the target.
 
       
-* `relation_to_epg` - (list) A list of Relation From NetFlow Exporter To EPG (ACI object [netflowRsExporterToEPg](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowRsExporterToEPg/overview)). This relation can point to multiple ACI objects:
+* `relation_from_netflow_exporter_to_epg` - (list) A list of Relation From NetFlow Exporter To EPG (ACI object [netflowRsExporterToEPg](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowRsExporterToEPg/overview)). This relation can point to multiple ACI objects:
```
Comment (sajagana) [inline docs/data-sources/netflow_exporter_policy.md:79]: This needs to be reverted.

**Review by sajagana — CHANGES_REQUESTED** [review]

File: docs/data-sources/l3out_consumer_label.md (hunk 2 above)
Comment (akinross) [reply docs/data-sources/l3out_consumer_label.md]: wrongly rebased changed in both datasource and resource

**Review by gmicol — APPROVED** [review]
LGTM

**Review by shrsr — APPROVED** [review]
LGTM

**Review by samiib — APPROVED** [review]
LGTM

**Review by sajagana — APPROVED** [review]
LGTM!

**Pull request merged**